src/services/userTransform.ts

```diff
@@ -231,6 +231,12 @@ export class UserTransformService {
       response.body = {
         error: extractStackTraceUptoLastSubstringMatch(error.stack, SUPPORTED_FUNC_NAMES),
       };
+      const metaTags = getTransformationMetadata(events[0]?.metadata);
```

Comment:
modify config-backend caller to reflect these tags.
/test call events metadata is constant and they do not have transformation or workspace details

Reply:
Or extract workspaceId from trRevcode which will have better hit rate